### DISCUSSION TITLE
Being able to connect to other osu!servers inside lazer login page

### DISCUSSION THREAD

**Discussion by zen1337th** [ppy/osu#24906] [Ideas] [answered]
I have thought of the idea being able to connect to a different osu server (for example like akatsuki.pw) in the login page inside lazer.
Personally I don't really like having thousands of shortcuts on my desktop to just play and connect to a different server, so I made a concept in Photoshop on how it could look like:
![image](https://github.com/ppy/osu/assets/104238285/f114a7d8-c8c8-45ea-a736-73be695a1f12)

**Comment by peppy** [comment, marked as answer]
Please read https://github.com/ppy/osu/discussions/18998.
